MenuList.java

```diff
@@ -0,0 +1,62 @@
+import java.util.*;
+/**
+ * This class is used to display the items available in the store
+ * @author Harshit
+ *
+ */
+public class MenuList {
+
+	public static List<Product> menul=new ArrayList<Product>();
+	public static void add()
+	{
+		menul=new ArrayList<Product>();
+		menul.add(new Product(1,"Apple        ",20,80));
+		menul.add(new Product(2,"Banana       ",10,30));
+		menul.add(new Product(3,"Kiwi         ",20,50));
+		menul.add(new Product(4,"UNO card     ",20,20));
+		menul.add(new Product(5,"Soft toy     ",30,400));
+		menul.add(new Product(6,"Towel        ",50,200));
+		menul.add(new Product(7,"Racing Car   ",40,1000));
+	}
+	/**
+	 * Displays the available items in the store
+	 */
+	public static void show()
+	{
+		System.out.println("Product Id"+"\t\t"+"Name"+"\t\t\t\t\t\t"+"Price"+"\t\t\t"+"Available Quantity");
+		for(Product menu1:menul)
+		{
+			System.out.println(menu1.getProductid()+"\t\t  "+menu1.getProductname()+"\t\t\t\t\t\t"+menu1.getProductprice()+"\t\t\t"+menu1.getProductquantity());
+		}
+	}
+	/**
+	 * Used to update the remaining quantity of items in the store
+	 * @param productid indicates product id
+	 * @param productquantity indicates product quantity after updation of cart
+	 */
+	public static void updatedQuantity(int productid,int productquantity)

```

Comment:
name should be updateQuantity(int productId, int productQuantity)

Product Id can be **Product7852** means not necessary an Integer